content/docs/hello-world.md

```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+在頁面上顯示 "Hello, world!"。
 
 [](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+按上面的連結會打開一個線上編輯器。歡迎把玩，看看輸出有什麼變化。本站的許多頁面都有像上面一樣可以編輯的範例。
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## 如何閱讀本指南 {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+本篇我們來看看 React 應用的基本組成：元素與組件。學好了這兩個，就可以打造小巧復用的組件，拼湊出複雜的應用。
```

Comment:
```diff
- 本篇我們來看看 React 應用的基本組成
+ 在這份指南中，我們會研究 React 應用程式的基本組成
```

```diff
- 學好了這兩個，就可以打造小巧復用的組件，拼湊出複雜的應用。
+ 一旦理解它們之後，你可以從小巧且可覆用的 Component，拼湊出複雜的應用程式。
```

The tech term `elements` and `components` maybe we can wait for our glossary and update it.